spec-2021x.md.html

```diff
@@ -924,7 +946,9 @@
 E_b(\mu_v) = \int_{0}^{2\pi} \int_{0}^{1} M_b(\mu_l, \phi, \mu_v) \, \mu_l \, \mathrm d \mu_l \, \mathrm d \phi
 \end{equation}
 
-![Figure [energy-compensation-sheen-e]: Directional albedo of sheen BRDF.](img/energycomp_sheen_e.png)
+![Figure [energy-compensation-sheen-e]: Directional albedo of the sheen BRDF.](img/energycomp_sheen_e.png)
+
+![Figure [white-furnace-sheen-e]: White furnace test of the sheen BRDF.](img/furnace_estevezkulla.png)
```

Comment:
The spheres are probably not necessary here, because the directional albedo image before already shows that the BRDF is energy conserving. There are a few red dots, perhaps because of the border handling or the low resolution.

I'd keep the spheres for the alternative sheen BRDF though, as it better visualizes the perceived error.

Reply:
Sure!